changelog

```diff
@@ -1,3 +1,477 @@
+        * Several date related conditions. (@warp-core)
+        * Conditions for the number of hyperspace jumps to another planet or system. (@warp-core)
+        * Flagship attribute. (@warp-core)
+        * Flagship landed. (@Hurleveur)
+        * Entered system by and previous system. (@Hurleveur, @Amazinite)
+      * Illegal outfits and cargo:
+        * "scan brightness" attribute, increases the chance an outfit is found on a scan. (@quyykk)
+        * "scan concealment" attribute, reduces the chance an outfit is found on a scan. (@quyykk)
+        * Illegal and attrocity goods can now be defined individually for different governments. (@Hurleveur, @tibetiroka)
+      * Added the option for a hyper, scram, or jump drive to have variable fuel cost based on the mass of the ship using it. (@Ferociousfeind)
+      * Added a new "merciful" personality that causes ships to avoid attacking hostile ships that are fleeing. (@Hurleveur, @quyykk)
+      * Added "drag reduction" and "inertia reduction" attributes. (@Azure3141)
+      * Systems can now define a "departure distance", a minimum distance a ship must be from the centre to jump to another system. (@RisingLeaf, @petervdmeer, @quyykk)
+      * When taking off from a planet with an outfitter while having more outfits in cargo than there is space for, store them instead of selling them. (@warp-core)
+      * Ship variant definitions can now remove all the bays from the base definition. (@daggs1)
+      * It is now possible to offer a mission in the shipyard or outfitter. (@rovermmicrover)
+      * A system definition can now include a custom value for the invisible fence radius. (@a-random-lemurian)
+      * TODO: find a concise way to describe the foreign penalty thing for governments. (@Hurleveur)
+      * Missions can now define a "to accept" condition set. (@rovermicrover)
+      * Governments can use named "stock" colors instead of always defining a color literal. (@warp-core)
+      * Added a "flotsam chance" attribute for outfits which determines the chance the outfit will be dropped as flotsam when the ship carrying it explodes. (@Azure3141)
+      * Governments can define custom values for various actions against foreign governments. (@warp-core)
+      * Added an "optical jamming" attribute that works similarly to radar jamming, but against optically guided missiles. (@Hurleveur)
+      * Added the "invisible" system property which makes systems inaccessible. (@yjhn, @Amazinite)
+      * Added a new "secretive" personality that causes a ship to fly away from ships attemmpting to scan it. (@Hurleveur)
+      * Added customizability to landing speeds. (@Hurleveur)
+      * Added more options for customizing ramscoop based fuel collection on a per system basis. (@Hurleveur)
+      * Added two new personalities: "daring" and "hunting" which separate the behaviors of "heroic" so they can be applied with more granularity. (@UnorderedSigh)
```

Comment:
We have multiple new personalities. Could just bundled them all into one section.